extensions/interactions/interaction_specs.json

```diff
@@ -97,6 +97,7 @@
     "default_outcome_heading": null,
     "rule_descriptions": {
       "MatchesExactlyWith": "matches exactly with {{x|NumericExpression}}",
+      "MatchesUpToTrivialManipulations": "matches up to trivial manipulations with {{x|NumericExpression}}",
```

Comment:
Maybe the human-readable description could be: "matches {{x|...}}, taking into account commutativity and associativity."

Otherwise no one knows what "trivial manipulations" means.

Ditto below.

Reply:
Done, thanks!